docs/environments.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Environments
+---
+
+# Environments
+
+## Descriptions
+
+Event Bus [Kafka](https://kafka.apache.org/) and applications are deployed into several environments for different purposes. The environments are described below.
+
+- **`Sandbox`** is a stable environment that external teams can connect to. It is a "customer dev" environment.
+- **`Pre-prod`** is a testing environment that has been cleared for PHI and PII. `Pre-prod` may be used for performance tests.
+- **`Prod`** is our production environment.
+
+## Deployment Schedule
+
+We deploy any new releases to all environments on Wednesdays.
+
+## Properties
+
+| Environment<br>Name | Kafka<br>endpoints                                                                                                                       | PHI, PII<br>Allowed |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| sandbox             | TBD                                                                                                                                      | No                  |
+| pre-prod            | TBD                                                                                                                                      | Yes                 |
+| prod                | b-1.eventbuskafka.m1c2oj.c3.kafka.us-gov-west-1.amazonaws.com:9098<br>b-2.eventbuskafka.m1c2oj.c3.kafka.us-gov-west-1.amazonaws.com:9098 | Yes                 |
```

Comment:
I might have misunderstood the takeaway in mob session, but I thought we had decided to remove the broker addresses since they're available in the catalog

Reply:
We talked about linking to the catalog, but decided against it. I thought we still wanted the broker addresses.

Reply:
I don't think these details are necessary to understand the environments we provide. If someone is interested in onboarding for a particular event, they'll be able to get those details from the catalog, which has the benefit of being restricted to the VA network. I'd vote for removing them, but I'm open to other thoughts from the team.

If we do keep them, FYI these brokers will be changed in the next couple of days since we're renaming our clusters